src/main/java/org/humancellatlas/ingest/project/ProjectService.java

```diff
@@ -175,4 +183,34 @@ private ProjectBag gather(Project project) {
         return new ProjectBag(projects, envelopes);
     }
 
+    public Page<Project> filterProjects(SearchFilter searchFilter, Pageable pageable) {
+        List<Criteria> criterias = new ArrayList<>();
+        criterias.add(Criteria.where("isUpdate").is(false));
+        Query query = new Query();
+
+        if (searchFilter.getWranglingState() != null) {
+            criterias.add(Criteria.where("wranglingState").is(searchFilter.getWranglingState()));
+        }
+
+        if (searchFilter.getWrangler() != null) {
+            criterias.add(Criteria.where("primaryWrangler").is(searchFilter.getWrangler()));
+        }
+
+        String search = searchFilter.getSearch();
+        if (search != null) {
+            criterias.add(new Criteria().orOperator(
+                    Criteria.where("content.project_core.project_title").regex(search, "i"),
+                    Criteria.where("content.project_core.project_description").regex(search, "i"),
+                    Criteria.where("content.project_core.project_short_name").regex(search, "i")
+            ));
+        }
+
+        query.addCriteria(new Criteria().andOperator(criterias.toArray(new Criteria[criterias.size()])));
```

Comment:
Quick latin lesson: 
Criteria is already plural. The singular is criterion. You can call the variable `criteria_list`
:)